EIPS/eip-6049.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 6049
+title: Deprecate SELFDESTRUCT
+description: Deprecate SELFDESTRUCT by discouraging its use and warning about a potential future behavior change.
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/deprecate-selfdestruct/11907
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
```suggestion
type: Meta
```
This doesn't require a hard fork, but makes non-optional recommendations. I think that meta is the correct EIP type.

Reply:
This requires text changes in the Yellow Paper.

Do other metas do that?

Reply:
Yes. Hard forks used to be Meta-EIPs.